config/s2i/jenkins/slave/Dockerfile

```diff
@@ -1,4 +1,7 @@
 FROM openshift/jenkins-slave-base-centos7:v3.7
 RUN yum install -y epel-release
+RUN yum install -y gcc python-devel libyaml-devel
```

Comment:
combine these RUN commands so we don't create so many layers in our docker images. And we should also yum -y clean all to clean up the database